### ISSUE TITLE
Handle no BLE adapter exception on windows

### ISSUE THREAD

**Issue by tdicola** [adafruit/adafruit-bluefruit-le-desktop#7]
Noble throws an error when no BLE adapter is present on windows.  Catch this and display the loading error page.

![image](https://cloud.githubusercontent.com/assets/1332183/9871254/3bd8427a-5b47-11e5-82c9-ea943e2b9187.png)

**Comment by d3drturner** [comment]
I get this error on windows 10 yet it claims to be connected
![image](https://cloud.githubusercontent.com/assets/14935103/10238769/24098f66-6892-11e5-9f4b-9092bb8f7b17.png)

**Comment by swstack** [comment]
@d3drturner did you install WinUSB drivers using the Zadig tool?

**Comment by d3drturner** [comment]
No, I installed the driver from here: https://www.adafruit.com/product/2661 
called PJRC CDC serial.  Not sure why I'd need another driver.  I can connect fine with arduino gui and VS 2015.

**Comment by tdicola** [comment]
Yeah you need to also install a libusb driver using Zadig tool, check out the steps here: https://github.com/adafruit/adafruit-bluefruit-le-desktop#windows  The issue is the noble library works around the spotty BLE support in Windows (mostly for older versions like Win 7) by talking directly to a USB BLE adapter.  In order to talk to the USB BLE adapter at a low level it has to be setup to use the libusb driver though, so make sure you're using the CSR8510 dongle and have used Zadig tool to assign it a libusb driver.  It's kind of a weird setup but only has to be done once for that device.  Thanks!
